src/components/TopologyPopover/topologySort.tsx

```diff
@@ -148,34 +148,38 @@ export const TopologySort = ({
     <>
       <div
         ref={popoverRef as LegacyRef<HTMLDivElement>}
-        className="flex mt-1 cursor-pointer md:mt-0 md:items-center border border-gray-300 bg-white rounded-md shadow-sm px-3 py-2"
+        className="flex cursor-pointer p-0 md:items-center border border-gray-300 bg-white rounded-md shadow-sm"
       >
-        {sortByDirection === "asc" && (
-          <BsSortUp
-            className="w-5 h-5 text-gray-700 hover:text-gray-900"
-            onClick={() => onSelectSortOption(sortBy, "desc")}
-          />
-        )}
-        {sortByDirection === "desc" && (
-          <BsSortDown
-            className="w-5 h-5 text-gray-700 hover:text-gray-900"
-            onClick={() => onSelectSortOption(sortBy, "asc")}
-          />
-        )}
-        <span
-          className="flex ml-2 text-sm text-gray-700 capitalize bold hover:text-gray-900"
+        <div
+          onClick={() =>
+            onSelectSortOption(
+              sortBy,
+              sortByDirection === "asc" ? "desc" : "asc"
+            )
+          }
+          className="flex items-center justify-center text-gray-700 hover:text-gray-900 px-3 py-1 pr-1"
+        >
+          {sortByDirection === "asc" && (
+            <BsSortUp className="w-5 h-5 text-gray-700 hover:text-gray-900" />
+          )}
+          {sortByDirection === "desc" && (
+            <BsSortDown className="w-5 h-5 text-gray-700 hover:text-gray-900" />
+          )}
+        </div>
+        <div
+          className="flex ml-2 text-sm text-gray-700 capitalize bold hover:text-gray-900 px-3 py-1 pl-0"
```

Comment:
The margin is not clickable
```suggestion
          className="flex text-sm text-gray-700 capitalize bold hover:text-gray-900 px-3 py-1"
```

Reply:
@moshloop  We need to set the padding to 0 on the left side in order to make the space in between look even. 

The margin issue should be better now.